pkg/app/cmd.go

```diff
@@ -164,12 +167,19 @@ func runE(
 }
 
 // the default is $ARTIFACTS if set, otherwise ./_artifacts
-func defaultArtifactsDir() string {
+// constructed as an absolute path to help the ginkgo tester because
+// for some reason it needs an absolute path to the kubeconfig
+func defaultArtifactsDir() (string, error) {
 	path, set := os.LookupEnv("ARTIFACTS")
 	if set {
-		return path
+		return path, nil
```

Comment:
nit: do we also want to abspath this?

Reply:
Done, also caused me to notice and fix a slight bug with the default.